SteamScript shortcuts does not work

Tried with `Space Engineers` and `Space Colony` and with each I get this error:
```
[ERROR] org.phoenicis.multithreading.ControlledThreadPoolExecutorService (l.64) - TypeError: com.oracle.truffle.api.interop.UnsupportedTypeException: Cannot convert '297920'(language: JavaScript, type: number) to Java type 'java.lang.String': Invalid or lossy primitive coercion.
	at <js> run(Unnamed:411:15584-15632)
	at <js> run(Unnamed:170:5379-5469)
	at <js> run(Unnamed:39-41:1232-1396)
	at <js> run(Unnamed:107:3183-3212)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.ObjectProxyHandler.invoke(HostInteropReflect.java:678)
	at com.sun.proxy.$Proxy44.run(Unknown Source)
	at org.phoenicis.library.ShortcutRunner.lambda$run$0(ShortcutRunner.java:52)
	at org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Exception in thread "pool-3-thread-4" TypeError: com.oracle.truffle.api.interop.UnsupportedTypeException: Cannot convert '297920'(language: JavaScript, type: number) to Java type 'java.lang.String': Invalid or lossy primitive coercion.
	at <js> run(Unnamed:411:15584-15632)
	at <js> run(Unnamed:170:5379-5469)
	at <js> run(Unnamed:39-41:1232-1396)
	at <js> run(Unnamed:107:3183-3212)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.ObjectProxyHandler.invoke(HostInteropReflect.java:678)
	at com.sun.proxy.$Proxy44.run(Unknown Source)
	at org.phoenicis.library.ShortcutRunner.lambda$run$0(ShortcutRunner.java:52)
	at org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

@ImperatorS79 I've moved past that error but D3DX9 verb gave me this error:
```
[ERROR] org.phoenicis.multithreading.ControlledThreadPoolExecutorService (l.64) - TypeError: <this>.architecture is not a function
	at <js> go(Unnamed:95:3384-3402)
	at <js> :anonymous(Unnamed:21:702-721)
	at <js> go(Unnamed:144:8014-8048)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.ObjectProxyHandler.invoke(HostInteropReflect.java:678)
	at com.sun.proxy.$Proxy44.go(Unknown Source)
	at org.phoenicis.javafx.components.application.skin.ApplicationInformationPanelSkin.lambda$installScript$7(ApplicationInformationPanelSkin.java:237)
	at org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Exception in thread "pool-3-thread-5" TypeError: <this>.architecture is not a function
	at <js> go(Unnamed:95:3384-3402)
	at <js> :anonymous(Unnamed:21:702-721)
	at <js> go(Unnamed:144:8014-8048)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.ObjectProxyHandler.invoke(HostInteropReflect.java:678)
	at com.sun.proxy.$Proxy44.go(Unknown Source)
	at org.phoenicis.javafx.components.application.skin.ApplicationInformationPanelSkin.lambda$installScript$7(ApplicationInformationPanelSkin.java:237)
	at org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

--- a/Engines/Wine/QuickScript/Steam Script/script.js
+++ b/Engines/Wine/QuickScript/Steam Script/script.js
@@ -1,12 +1,14 @@
 const QuickScript = include("engines.wine.quick_script.quick_script");
 const Downloader = include("utils.functions.net.download");
 const Wine = include("engines.wine.engine.object");
-const {cat, fileExists, writeToFile, createTempFile} = include("utils.functions.filesystem.files");
+const { cat, fileExists, writeToFile, createTempFile } = include("utils.functions.filesystem.files");
 
 const Luna = include("engines.wine.verbs.luna");
 const Corefonts = include("engines.wine.verbs.corefonts");
 include("engines.wine.plugins.override_dll");
 include("engines.wine.plugins.windows_version");
+
+const Thread = Java.type("java.lang.Thread");
 
 module.default = class SteamScript extends QuickScript {
     constructor() {
@@ -36,30 +38,44 @@ module.default = class SteamScript extends QuickScript {
     manifest(wine) {
         if (!this._manifest) {
             // cache manifest path (will not change during the installation)
-            this._manifest = wine.prefixDirectory() + "/drive_c/" + wine.programFiles() + "/Steam/steamapps/appmanifest_" + this._appId + ".acf";
+            this._manifest = `${wine.prefixDirectory()}/drive_c/${wine.programFiles()}/Steam/steamapps/appmanifest_${this._appId}.acf`;
         }
+
         return this._manifest;
     }
 
+    /**
+     * Checks whether the software download has started
+     *
+     * @param {Wine} wine The wine engine object
+     * @returns {boolean} True if the download has started, false otherwise
+     */
     downloadStarted(wine) {
         if (fileExists(this.manifest(wine))) {
             const manifest = cat(this.manifest(wine));
-            const state = Number(manifest.match(/\"StateFlags\"\s+\"(\d+)\"/)[1]);
+            const state = parseInt(manifest.match(/"StateFlags"\s+"(\d+)"/)[1], 10);
+
             return state == 1026 || state == 1042 || state == 1062 || state == 1030;
-        }
-        else {
+        } else {
             return false;
         }
     }
 
+    /**
+     * Checks whether the software download has finished.
+     * To get a valid result with this method `downloadStarted(wine)` needs to return `true` first
+     *
+     * @param {Wine} wine The wine engine object
+     * @returns {boolean} True if the download has finished, false otherwise
+     */
     downloadFinished(wine) {
         // check if download already finished (download folder has been deleted)
         if (fileExists(this.manifest(wine))) {
             const manifest = cat(this.manifest(wine));
-            const state = Number(manifest.match(/\"StateFlags\"\s+\"(\d+)\"/)[1]);
+            const state = parseInt(manifest.match(/"StateFlags"\s+"(\d+)"/)[1], 10);
+
             return state != 1026 && state != 1042 && state != 1062 && state != 1030;
-        }
-        else {
+        } else {
             return false;
         }
     }
@@ -67,8 +83,9 @@ module.default = class SteamScript extends QuickScript {
     configVdf(wine) {
         if (!this._configVdf) {
             // cache config.vdf path (will not change during the installation)
-            this._configVdf = wine.prefixDirectory() + "/drive_c/" + wine.programFiles() + "/Steam/config/config.vdf";
+            this._configVdf = `${wine.prefixDirectory()}/drive_c/${wine.programFiles()}/Steam/config/config.vdf`;
         }
+
         return this._configVdf;
     }
 
@@ -76,21 +93,22 @@ module.default = class SteamScript extends QuickScript {
     fixCertificateIssue(wine) {
         const steamConfigFile = this.configVdf(wine);
         const steamConfig = cat(steamConfigFile);
-        const cmPos = steamConfig.indexOf("\"CM\"");
-        const csConfig = "\"CS\" \"valve511.steamcontent.com;valve501.steamcontent.com;valve517.steamcontent.com;valve557.steamcontent.com;valve513.steamcontent.com;valve535.steamcontent.com;valve546.steamcontent.com;valve538.steamcontent.com;valve536.steamcontent.com;valve530.steamcontent.com;valve559.steamcontent.com;valve545.steamcontent.com;valve518.steamcontent.com;valve548.steamcontent.com;valve555.steamcontent.com;valve556.steamcontent.com;valve506.steamcontent.com;valve544.steamcontent.com;valve525.steamcontent.com;valve567.steamcontent.com;valve521.steamcontent.com;valve510.steamcontent.com;valve542.steamcontent.com;valve519.steamcontent.com;valve526.steamcontent.com;valve504.steamcontent.com;valve500.steamcontent.com;valve554.steamcontent.com;valve562.steamcontent.com;valve524.steamcontent.com;valve502.steamcontent.com;valve505.steamcontent.com;valve547.steamcontent.com;valve560.steamcontent.com;valve503.steamcontent.com;valve507.steamcontent.com;valve553.steamcontent.com;valve520.steamcontent.com;valve550.steamcontent.com;valve531.steamcontent.com;valve558.steamcontent.com;valve552.steamcontent.com;valve563.steamcontent.com;valve540.steamcontent.com;valve541.steamcontent.com;valve537.steamcontent.com;valve528.steamcontent.com;valve523.steamcontent.com;valve512.steamcontent.com;valve532.steamcontent.com;valve561.steamcontent.com;valve549.steamcontent.com;valve522.steamcontent.com;valve514.steamcontent.com;valve551.steamcontent.com;valve564.steamcontent.com;valve543.steamcontent.com;valve565.steamcontent.com;valve529.steamcontent.com;valve539.steamcontent.com;valve566.steamcontent.com;valve165.steamcontent.com;valve959.steamcontent.com;valve164.steamcontent.com;valve1611.steamcontent.com;valve1601.steamcontent.com;valve1617.steamcontent.com;valve1603.steamcontent.com;valve1602.steamcontent.com;valve1610.steamcontent.com;valve1615.steamcontent.com;valve909.steamcontent.com;valve900.steamcontent.com;valve905.steamcontent.com;valve954.steamcontent.com;valve955.steamcontent.com;valve1612.steamcontent.com;valve1607.steamcontent.com;valve1608.steamcontent.com;valve1618.steamcontent.com;valve1619.steamcontent.com;valve1606.steamcontent.com;valve1605.steamcontent.com;valve1609.steamcontent.com;valve907.steamcontent.com;valve901.steamcontent.com;valve902.steamcontent.com;valve1604.steamcontent.com;valve908.steamcontent.com;valve950.steamcontent.com;valve957.steamcontent.com;valve903.steamcontent.com;valve1614.steamcontent.com;valve904.steamcontent.com;valve952.steamcontent.com;valve1616.steamcontent.com;valve1613.steamcontent.com;valve958.steamcontent.com;valve956.steamcontent.com;valve906.steamcontent.com\"\n";
+        const cmPos = steamConfig.indexOf(`"CM"`);
+        const csConfig = `"CS" "valve511.steamcontent.com;valve501.steamcontent.com;valve517.steamcontent.com;valve557.steamcontent.com;valve513.steamcontent.com;valve535.steamcontent.com;valve546.steamcontent.com;valve538.steamcontent.com;valve536.steamcontent.com;valve530.steamcontent.com;valve559.steamcontent.com;valve545.steamcontent.com;valve518.steamcontent.com;valve548.steamcontent.com;valve555.steamcontent.com;valve556.steamcontent.com;valve506.steamcontent.com;valve544.steamcontent.com;valve525.steamcontent.com;valve567.steamcontent.com;valve521.steamcontent.com;valve510.steamcontent.com;valve542.steamcontent.com;valve519.steamcontent.com;valve526.steamcontent.com;valve504.steamcontent.com;valve500.steamcontent.com;valve554.steamcontent.com;valve562.steamcontent.com;valve524.steamcontent.com;valve502.steamcontent.com;valve505.steamcontent.com;valve547.steamcontent.com;valve560.steamcontent.com;valve503.steamcontent.com;valve507.steamcontent.com;valve553.steamcontent.com;valve520.steamcontent.com;valve550.steamcontent.com;valve531.steamcontent.com;valve558.steamcontent.com;valve552.steamcontent.com;valve563.steamcontent.com;valve540.steamcontent.com;valve541.steamcontent.com;valve537.steamcontent.com;valve528.steamcontent.com;valve523.steamcontent.com;valve512.steamcontent.com;valve532.steamcontent.com;valve561.steamcontent.com;valve549.steamcontent.com;valve522.steamcontent.com;valve514.steamcontent.com;valve551.steamcontent.com;valve564.steamcontent.com;valve543.steamcontent.com;valve565.steamcontent.com;valve529.steamcontent.com;valve539.steamcontent.com;valve566.steamcontent.com;valve165.steamcontent.com;valve959.steamcontent.com;valve164.steamcontent.com;valve1611.steamcontent.com;valve1601.steamcontent.com;valve1617.steamcontent.com;valve1603.steamcontent.com;valve1602.steamcontent.com;valve1610.steamcontent.com;valve1615.steamcontent.com;valve909.steamcontent.com;valve900.steamcontent.com;valve905.steamcontent.com;valve954.steamcontent.com;valve955.steamcontent.com;valve1612.steamcontent.com;valve1607.steamcontent.com;valve1608.steamcontent.com;valve1618.steamcontent.com;valve1619.steamcontent.com;valve1606.steamcontent.com;valve1605.steamcontent.com;valve1609.steamcontent.com;valve907.steamcontent.com;valve901.steamcontent.com;valve902.steamcontent.com;valve1604.steamcontent.com;valve908.steamcontent.com;valve950.steamcontent.com;valve957.steamcontent.com;valve903.steamcontent.com;valve1614.steamcontent.com;valve904.steamcontent.com;valve952.steamcontent.com;valve1616.steamcontent.com;valve1613.steamcontent.com;valve958.steamcontent.com;valve956.steamcontent.com;valve906.steamcontent.com"\n`;
         const newSteamConfig = steamConfig.slice(0, cmPos) + csConfig + steamConfig.slice(cmPos);
-        writeToFile(steamConfigFile, newSteamConfig)
+
+        writeToFile(steamConfigFile, newSteamConfig);
     }
 
     go() {
         // default application homepage if not specified
         if (!this._applicationHomepage) {
-            this._applicationHomepage = "https://store.steampowered.com/app/" + this._appId;
+            this._applicationHomepage = `https://store.steampowered.com/app/${this._appId}`;
         }
 
         // default executable args if not specified
         if (!this._executableArgs) {
-            this._executableArgs = ["-no-cef-sandbox", "-silent", "-applaunch", this._appId];
+            this._executableArgs = ["-no-cef-sandbox", "-silent", "-applaunch", `${this._appId}`];
         }
 
         const setupWizard = SetupWizard(InstallationType.APPS, this._name, this.miniature());
@@ -114,23 +132,36 @@ module.default = class SteamScript extends QuickScript {
         new Corefonts(wine).go();
 
         // Steam must be started once such that config.vdf is created (see fixCertificateIssue())
-        setupWizard.wait(tr("Please follow the steps of the Steam setup. Then, wait until Steam is updated, log in and finally close Steam completely so the installation of \"{0}\" can continue.", this._name));
+        setupWizard.wait(
+            tr(
+                'Please follow the steps of the Steam setup. Then, wait until Steam is updated, log in and finally close Steam completely so the installation of "{0}" can continue.',
+                this._name
+            )
+        );
         wine.run(tempFile, [], null, false, true);
 
         // Set windows environment for executable that needs it
-        wine.setOsForApplication().set("steam.exe", "winxp").do();
-        wine.setOsForApplication().set("steamwebhelper.exe", "winxp").do();
+        wine.setOsForApplication()
+            .set("steam.exe", "winxp")
+            .do();
+        wine.setOsForApplication()
+            .set("steamwebhelper.exe", "winxp")
+            .do();
 
         // Fix for Uplay games that are executed on steam
-        wine.setOsForApplication().set("upc.exe", "winvista").do();
-        wine.setOsForApplication().set("UbisoftGameLauncher.exe", "winvista").do();
+        wine.setOsForApplication()
+            .set("upc.exe", "winvista")
+            .do();
+        wine.setOsForApplication()
+            .set("UbisoftGameLauncher.exe", "winvista")
+            .do();
 
         // ensure that Steam is running (user might have unchecked "run Steam after installation finished")
-        wine.runInsidePrefix(wine.programFiles() + "/Steam/Steam.exe", ["steam://nav/games"], false);
+        wine.runInsidePrefix(`${wine.programFiles()}/Steam/Steam.exe`, ["steam://nav/games"], false);
 
         // wait until config.vdf exists
         while (!fileExists(this.configVdf(wine))) {
-            java.lang.Thread.sleep(1000);
+            Thread.sleep(1000);
         }
 
         // wait until Steam and Wine are closed
@@ -146,22 +177,22 @@ module.default = class SteamScript extends QuickScript {
         // back to generic wait (might have been changed in preInstall)
         setupWizard.wait(tr("Please wait..."));
 
-        wine.runInsidePrefix(wine.programFiles() + "/Steam/Steam.exe", ["steam://install/" + this._appId], false);
+        wine.runInsidePrefix(`${wine.programFiles()}/Steam/Steam.exe`, [`steam://install/${this._appId}`], false);
 
         setupWizard.wait(tr("Please wait until Steam has finished the download..."));
 
         // wait until download started
         while (!this.downloadStarted(wine)) {
-            java.lang.Thread.sleep(100);
+            Thread.sleep(100);
         }
 
         // make sure download is finished
         while (!this.downloadFinished(wine)) {
-            java.lang.Thread.sleep(1000);
+            Thread.sleep(1000);
         }
 
         // close Steam
-        wine.runInsidePrefix(wine.programFiles() + "/Steam/Steam.exe", "-shutdown", true);
+        wine.runInsidePrefix(`${wine.programFiles()}/Steam/Steam.exe`, "-shutdown", true);
 
         // back to generic wait
         setupWizard.wait(tr("Please wait..."));
@@ -183,4 +214,4 @@ module.default = class SteamScript extends QuickScript {
 
         setupWizard.close();
     }
-}
+};

--- a/Engines/Wine/Verbs/D9VK/script.js
+++ b/Engines/Wine/Verbs/D9VK/script.js
@@ -34,7 +34,7 @@ class D9VK {
         const wizard = this.wine.wizard();
         const prefixDirectory = this.wine.prefixDirectory();
         const system32directory = this.wine.system32directory();
-        const architecture = this.wine.architecture():
+        const architecture = this.wine.architecture();
 
         print("NOTE: Wine version should be greater or equal to 3.10");
 

--- a/Engines/Wine/Verbs/gdiplus/script.js
+++ b/Engines/Wine/Verbs/gdiplus/script.js
@@ -27,15 +27,15 @@ class GDIPlus {
             .checksum("c3516bc5c9e69fee6d9ac4f981f5b95977a8a2fa")
             .name("windows6.1-KB976932-X86.exe")
             .get();
-      
+
         new CabExtract()
             .archive(setupFile)
 	    .wizard(wizard)
             .to(`${prefixDirectory}/drive_c/gdiplus/`)
             .extract(["-L", "-F", "x86_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.17514_none_72d18a4386696c80/gdiplus.dll"]);
-		
+
         cp(`${prefixDirectory}/drive_c/gdiplus/x86_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.17514_none_72d18a4386696c80/gdiplus.dll`, system32directory);
-	
+
         if (architecture == "amd64") {
 	    const system64directory = this.wine.system64directory();
 
@@ -45,13 +45,13 @@ class GDIPlus {
                 .checksum("74865ef2562006e51d7f9333b4a8d45b7a749dab")
                 .name("windows6.1-KB976932-X64.exe")
                 .get();
-			
+
             new CabExtract()
                 .archive(setupFile64)
-		.wizard(wizard)
+                .wizard(wizard)
                 .to(`${prefixDirectory}/drive_c/gdiplus/`)
                 .extract(["-L", "-F", "amd64_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.17514_none_2b24536c71ed437a/gdiplus.dll"]);
-		
+
             cp(`${prefixDirectory}/drive_c/gdiplus/amd64_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.17514_none_2b24536c71ed437a/gdiplus.dll`, system64directory);
         }
 

--- a/Engines/Wine/Verbs/gdiplus/script.js
+++ b/Engines/Wine/Verbs/gdiplus/script.js
@@ -30,14 +30,14 @@ class GDIPlus {
 
         new CabExtract()
             .archive(setupFile)
-	    .wizard(wizard)
+            .wizard(wizard)
             .to(`${prefixDirectory}/drive_c/gdiplus/`)
             .extract(["-L", "-F", "x86_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.17514_none_72d18a4386696c80/gdiplus.dll"]);
 
         cp(`${prefixDirectory}/drive_c/gdiplus/x86_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.17514_none_72d18a4386696c80/gdiplus.dll`, system32directory);
 
         if (architecture == "amd64") {
-	    const system64directory = this.wine.system64directory();
+            const system64directory = this.wine.system64directory();
 
             const setupFile64 = new Resource()
                 .wizard(wizard)

--- a/Engines/Wine/Verbs/gdiplus_winxp/script.js
+++ b/Engines/Wine/Verbs/gdiplus_winxp/script.js
@@ -31,13 +31,13 @@ class GDIPlusWinXP {
 
         new CabExtract()
             .archive(setupFile)
-	    .wizard(wizard)
+            .wizard(wizard)
             .to(`${prefixDirectory}/drive_c/gdiplus/`)
             .extract(["-F", "FL_gdiplus_dll_____X86.3643236F_FC70_11D3_A536_0090278A1BB8"]);
 
         new CabExtract()
             .archive(setupFile)
-	    .wizard(wizard)
+            .wizard(wizard)
             .to(`${prefixDirectory}/drive_c/gdiplus/`)
             .extract(["-L", "-F", "x86_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.17514_none_72d18a4386696c80/gdiplus.dll"]);
 
@@ -65,4 +65,4 @@ class GDIPlusWinXP {
     }
 }
 
-module.default = GDIPlusWindowsXP;
+module.default = GDIPlusWinXP;

--- a/Engines/Wine/Verbs/gdiplus_winxp/script.js
+++ b/Engines/Wine/Verbs/gdiplus_winxp/script.js
@@ -28,22 +28,22 @@ class GDIPlusWinXP {
             .checksum("6113cd89d77525958295ccbd73b5fb8b89abd0aa")
             .name("NDP1.0sp2-KB830348-X86-Enu.exe")
             .get();
-			
+
         new CabExtract()
             .archive(setupFile)
 	    .wizard(wizard)
             .to(`${prefixDirectory}/drive_c/gdiplus/`)
             .extract(["-F", "FL_gdiplus_dll_____X86.3643236F_FC70_11D3_A536_0090278A1BB8"]);
-      
+
         new CabExtract()
             .archive(setupFile)
 	    .wizard(wizard)
             .to(`${prefixDirectory}/drive_c/gdiplus/`)
             .extract(["-L", "-F", "x86_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.17514_none_72d18a4386696c80/gdiplus.dll"]);
-			
+
         const content = cat(`${prefixDirectory}/drive_c/gdiplus/drive_c/gdiplus/FL_gdiplus_dll_____X86.3643236F_FC70_11D3_A536_0090278A1BB8`);
         writeToFile(`${system32directory}/gdiplus.dll`, content);
-		
+
         remove(`${prefixDirectory}/drive_c/gdiplus/`);
 
         this.wine

--- a/Engines/Wine/Verbs/xact/script.js
+++ b/Engines/Wine/Verbs/xact/script.js
@@ -197,7 +197,7 @@ class Xact {
 
         if (architecture == "amd64") {
             const system64directory = this.wine.system64directory();
-            
+
             //---------------------------------------------------------Extract xactengine*.dll (x64)--------------------------------------------
             new CabExtract()
                 .wizard(wizard)

